src/python/bot/fuzzers/engine_common.py

```diff
@@ -452,6 +472,10 @@ def get_all_issue_metadata(fuzz_target_path):
   if issue_owners:
     metadata['issue_owners'] = ','.join(issue_owners)
 
+  additional_issue_fields = get_additional_fields(fuzz_target_path)
+  if additional_issue_fields:
```

Comment:
Let's do this in a way that's more extensible in the future. We can make the format of the ".issue_metadata" file be a json that looks like this:

```
{
  "additional_fields": ...,
}
```

That way, if we need to add additional metadata, we can just add to this json object without breaking the format.